packages/oapp-evm-upgradeable/artifacts/ExecutorOptions.sol/ExecutorOptions.json

```diff
@@ -5,6 +5,7 @@
       "name": "Executor_InvalidLzComposeOption",
       "inputs": []
     },
+    { "type": "error", "name": "Executor_InvalidLzReadOption", "inputs": [] },
```

Comment:
I believe so, the artifact is just from the fact that ExecutorOptions have been changed protocol wide (so that there is now an `ExecutorLzReadOption`:

https://github.com/LayerZero-Labs/LayerZero-v2/blob/943ce4a2bbac070f838e12c7fd034bca6a281ccf/packages/layerzero-v2/evm/messagelib/contracts/libs/ExecutorOptions.sol#L21

Believe this is just being picked up in the artifacts now when running `pnpm build` @ryandgoulding.